src/scripts/Viewer.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+	<div class="wrapper">
+		<div class="viewer">
+			<button class="viewer--close" @click="close"></button>
+			<div class="viewer--container" :style="'background-image:url(' + imgSrc+ ')'"></div>
+		</div>
+	</div>
+</template>
+
+<script>
+const appName = require('../../package.json').name;
+
+export default {
+	name: "Viewer",
+	methods: {
+		close() {
+			this.$router.push('/');
+		}
+	},
+	computed: {
+		file() {
+			return _.findWhere(FileList.files, {
+				name: this.$route.params.file
+			});
+		},
+		dir() {
+			return window[appName].context.dir;
+		},
+		list() {
+			return _.filter(FileList.files, {
+				mimetype: "image/png"
+			});
+		},
+		thumbDimensions() {
+			let width = $(window).width();
+			switch (true) {
+				case (width <= 1024) :
```

Comment:
will there be more in the near future ? what about 8k TVs ? ;-)

Reply:
I'll go up to 4K … aight? ;-)